### PR TITLE
Rentals list

### PR DESCRIPTION
**Pull request by adrian-ponce-wolox**
## Summary ##
**Trello card**: https://trello.com/c/WhvZvFzW/26-vista-para-libros-alquilados
**Trello card**: https://trello.com/c/widheS0P/25-wishlist

Whislist view and Rentals view

## Test Plan ##

  * Open the app
  * Login with Google
  * The main tab bar view should appear after a successfull login.
  * Press on Rentals tab
  * My Rentals tab should appear and load book cells
  * Press on Wishes tab
  * Wishes tab should appear and load book cells

### iPhone 7

rentals

whises

## Screencast ##

rentals_wishes_list

### CODE REVIEW

File: WBooks/BookDetail/BookDetailViewModel.swift [hunk 1]
```diff
@@ -15,11 +15,13 @@ final class BookDetailViewModel {
     private let _booksRepository: BooksRepository
     private let _userRepository: UserRepository
     private(set) var _bookViewModel: BookViewModel
+    private let _originTableType: BookTableType
```
Comment (danielaRiesgo) [inline WBooks/BookDetail/BookDetailViewModel.swift]: If you are not using this for anything, why are you storing it?

File: WBooks/BookTable/BooksTableViewModel.swift [hunk 2]
```diff
@@ -9,25 +9,48 @@
...
-            .on(next: { [unowned self] books in self._viewModels = books.map { BookViewModel(book: $0) } })
-            .map { _ in return }
+        switch _bookTableType {
+        case .Library:
+            return _booksRepository.fetchBooks()
+                .on(next: { [unowned self] books in self._viewModels = books.map { BookViewModel(book: $0) } })
+                .map { _ in return }
+        case .Wishlist:
+            return _userRepository.fetchWishes()
+                .on(next: { [unowned self] books in self._viewModels = books.map { wish in
```
Comment (danielaRiesgo) [inline WBooks/BookTable/BooksTableViewModel.swift]: You can do as above in one line:
`.on(next: { [unowned self] books in self._viewModels = books.map { BookViewModel(book: $0.book) }`

File: WBooks/BookTable/BooksTableViewModel.swift [hunk 3]
```diff
@@ -9,25 +9,48 @@
...
+                    return BookViewModel(book: wish.book)
+                    } })
+                .map { _ in return }
+        case .Rentals:
+            return _userRepository.fetchRentals()
+                .on(next: { [unowned self] rents in self._viewModels = rents.map { rent in
+                    return BookViewModel(book: rent.book)
```
Comment (danielaRiesgo) [inline WBooks/BookTable/BooksTableViewModel.swift]: You can do as above in one line:
`.on(next: { [unowned self] books in self._viewModels = books.map { BookViewModel(book: $0.book) }`

File: Localization/es.lproj/Localizable.strings [hunk 4]
```diff
@@ -22,6 +22,12 @@
 "HOME_TAB.SUGGEST" = "Sugeridos";
 "HOME_TAB.RENTALS" = "Alquileres";
 
+// BOOK_TABLE
+"BOOK_TABLE.LIBRARY_TITLE" = "BIBLIOTECA";
+"BOOK_TABLE.WISHLIST_TITLE" = "DESEADOS";
+"BOOK_TABLE.SUGGEST_TITLE" = "SUGERIR LIBRO";
+"BOOK_TABLE.RENTALS_TITLE" = "MIS ALQUILADOS";
```
Comment (dtilve) [inline Localization/es.lproj/Localizable.strings]: `"ALQUILERES"`?

**Pull request merged**